sources/legal/latest/policies/acceptable-use-policy.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Apify Acceptable Use Policy
+description: Apify's acceptable use policy describes activities that are prohibited on the Apify platform and on our websites.
+sidebar_label: Acceptable Use Policy
+sidebar_position: 1
+category: legal
+slug: /acceptable-use-policy
+---
+
+<!-- vale off -->
+
+# Apify Acceptable Use Policy
```

Comment:
If h1 is the same as `title` it doesn't make sense to have both, `title`. I would suggest removing H1 especially since it seems to be rendering twice at the moment 

![obraz](https://github.com/apify/apify-docs/assets/92638966/e58eb889-7baf-4b7c-a2c8-7e6d059226ed)